src/quotamodel.c

```diff
@@ -2118,6 +2120,10 @@ refresh_rejectmap(PG_FUNCTION_ARGS)
 			}
 			LWLockRelease(diskquota_locks.relation_cache_lock);
 		}
+		if (HeapTupleIsValid(tuple))
```

Comment:
Why the second `if (HeapTupleIsValid(tuple))` has been added? Can we call heap_freetuple for tuple at the end of the positive branch of the first `if (HeapTupleIsValid(tuple))`?